client/coral-admin/src/components/UserDetail.css

```diff
@@ -149,6 +149,12 @@
   color: white;
 }
 
+.actionsMenuAlwaysPremod {
```

Comment:
If you move this above line `140: .actionsMenuSuspended`, then _suspension_ and _banned_ styles will have higher priority than _alwaysPremod_